localstack-core/localstack/services/apigateway/next_gen/execute_api/context.py

```diff
@@ -33,55 +35,21 @@ class InvocationRequest(TypedDict, total=False):
-
-
-class IdentityContext(TypedDict, total=False):
-    # https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-mapping-template-reference.html
-    accountId: Optional[str]
-    """The AWS account ID associated with the request."""
-    apiKey: Optional[str]
-    """For API methods that require an API key, this variable is the API key associated with the method request."""
-    apiKeyId: Optional[str]
-    """The API key ID associated with an API request that requires an API key."""
-    caller: Optional[str]
-    """The principal identifier of the caller that signed the request. Supported for resources that use IAM authorization."""
-    cognitoAuthenticationProvider: Optional[str]
-    """A comma-separated list of the Amazon Cognito authentication providers used by the caller making the request"""
-    cognitoAuthenticationType: Optional[str]
-    """The Amazon Cognito authentication type of the caller making the request"""
-    cognitoIdentityId: Optional[str]
-    """The Amazon Cognito identity ID of the caller making the request"""
-    cognitoIdentityPoolId: Optional[str]
-    """The Amazon Cognito identity pool ID of the caller making the request"""
-    principalOrgId: Optional[str]
-    """The AWS organization ID."""
-    sourceIp: Optional[str]
-    """The source IP address of the immediate TCP connection making the request to the API Gateway endpoint"""
-    clientCert: Optional[dict[str, str]]
-    """Certificate that a client presents. Present only in access logs if mutual TLS authentication fails."""
-    vpcId: Optional[str]
-    """The VPC ID of the VPC making the request to the API Gateway endpoint."""
-    vpceId: Optional[str]
-    """The VPC endpoint ID of the VPC endpoint making the request to the API Gateway endpoint."""
-    user: Optional[str]
-    """The principal identifier of the user that will be authorized against resource access for resources that use IAM authorization."""
-    userAgent: Optional[str]
-    """The User-Agent header of the API caller."""
-    userArn: Optional[str]
-    """The Amazon Resource Name (ARN) of the effective user identified after authentication."""
-
-
-class ContextVariables(TypedDict, total=False):
-    authorizer: AuthorizerContext
-    identity: IdentityContext
+class IntegrationRequest(TypedDict, total=False):
+    http_method: Optional[HTTPMethod]
+    """HTTP Method of the incoming request"""
+    uri: Optional[str]
+    """URI of the integration"""
+    query_string_parameters: Optional[dict[str, str]]
+    """Query string parameters of the request"""
+    headers: Optional[dict[str, str]]
+    """Headers of the request"""
+    multi_value_query_string_parameters: Optional[dict[str, list[str]]]
+    """Multi value query string parameters of the request"""
+    multi_value_headers: Optional[dict[str, list[str]]]
+    """Multi value headers of the request"""
+    body: Optional[bytes]
+    """Body content of the request"""
```

Comment:
Having that clear separation from the invocation request will be key in maintaining transparency over the requests.